Neuro/Minigames/MinigameHandler.cs

```diff
@@ -41,8 +41,12 @@ private static IEnumerator CoTryCompleteMinigame(Minigame minigame, PlayerTask t
         InGameCursor.Instance.HideWhen(() => !minigame);
 
         yield return new WaitForSeconds(0.4f);
+        float startTime = Time.time;
         yield return solver.CompleteMinigame(minigame, task);
-
+        
+        float timeDiff = Time.time - startTime;
+        MinigameTimeHandler.Instance.AddMinigameTime(minigame, timeDiff);
```

Comment:
Since the solver might quit before the minigame is actually done, it might be better to instead wait for Minigame.Instance to be null before adding the time.

Reply:
This should be addressed now.